rskj-core/src/main/java/co/rsk/peg/StateForFederator.java

```diff
@@ -18,37 +18,32 @@
 
 package co.rsk.peg;
 
-import co.rsk.crypto.Keccak256;
-import co.rsk.bitcoinj.core.NetworkParameters;
 import co.rsk.bitcoinj.core.BtcTransaction;
-import org.ethereum.util.RLP;
-import org.ethereum.util.RLPList;
-
+import co.rsk.crypto.Keccak256;
+import java.util.Collections;
 import java.util.SortedMap;
+import org.ethereum.util.RLP;
 
-/**
- * Created by mario on 20/04/17.
- */
 public class StateForFederator {
-    private SortedMap<Keccak256, BtcTransaction> rskTxsWaitingForSignatures;
 
-    public StateForFederator(SortedMap<Keccak256, BtcTransaction> rskTxsWaitingForSignatures) {
-        this.rskTxsWaitingForSignatures = rskTxsWaitingForSignatures;
-    }
+    private final SortedMap<Keccak256, BtcTransaction> rskTxsWaitingForSignatures;
 
-    public StateForFederator(byte[] rlpData, NetworkParameters parameters) {
```

Comment:
Careful careful, I believe this is used from the powpeg-node

Reply:
Okay it makes sense now, will update :) Thanks for the heads up